skeleton/local-deploy/ai-lab.yaml

```diff
@@ -0,0 +1,27 @@
+version: v1.0
+application:
+  type: language
+  name: AI Application
+  description: AI Application Being Locally Deployed
+  containers:
+    - name: llamacpp-server
+      contextdir: ./deployment/model_servers/llamacpp_python
+      containerfile: ./base/Containerfile
+      model-service: true
+      backend:
+        - llama
+      arch:
+        - arm64
+        - amd64
+      ports:
+        - 8001
+      image: ${{ values.imageRegistry }}/${{ values.imageOrg }}/llamacpp_python:latest
```

Comment:
Do we need to also build the model image? I think we could probably get away with using a fixed image here?